docs/source/conf.py

```diff
@@ -242,7 +242,42 @@ def render_svg_logo(path):
         os.remove(header_file)
 
 
+def copy_example_assets(app, exception):
```

Comment:
why was the symlink failed to work?

Reply:
You're right! I've updated `build.sh` to remove asset copying